sqladmin/models.py

```diff
@@ -525,7 +631,17 @@ async def update_model(self, pk: Any, data: Dict[str, Any]) -> None:
             await anyio.to_thread.run_sync(self._update_modeL_sync, pk, data)
 
     async def scaffold_form(self) -> Type[Form]:
-        return await get_model_form(model=self.model, engine=self.engine)
+        if self.form is not None:
+            return self.form
```

Comment:
Coverage missing here.

Reply:
I don't have the right permissions, I tried to add the test:

Reply:
I don't see what test you were trying to add, but I added one called `test_form_override_scaffold` inside `test_forms.py`. Good catch on lack of coverage.

Reply:
Yes, that was exactly it.